ui/src/app/utils/subscribable.tsx

```diff
@@ -0,0 +1,43 @@
+import * as fp from 'lodash/fp';
+
```

Comment:
Some simple unit tests around the Atom to codify some of its more nuanced behaviors could be nice: synchronous vs asynchronous behaviors, subscriptions, behavior on initial data seeding